### PR TITLE
Exporter cleanup and command line utility

### PR DESCRIPTION
**Pull request by gramalingam**
Exporter parameter cleanup and command line utility

### CODE REVIEW

File: tools/onnx2script.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# -------------------------------------------------------------------------
```
Comment (justinchuby) [inline tools/onnx2script.py:1]: I would add usage in the module docstring

File: tools/onnx2script.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    with open(output_file_name, "w", encoding="utf-8") as f:
+        f.write(python_code)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Convert ONNX model file to onnxscript file")
+    parser.add_argument("input", help="ONNX model file to convert")
+    parser.add_argument("-o", "--output", help="Output file name")
+    parser.add_argument(
+        "-v", "--verbose", action="store_true", help="Verbose mode", default=False
```
Comment (justinchuby) [inline tools/onnx2script.py:37]: I would include what this mode does?

**Review by justinchuby — APPROVED** [review]

File: tools/onnx2script.py (hunk 1 above)
Comment (gramalingam) [reply tools/onnx2script.py:1]: Done

File: tools/onnx2script.py (hunk 2 above)
Comment (gramalingam) [reply tools/onnx2script.py]: Done

**Pull request merged**